packages/calcite-components/src/components/tab-nav/tab-nav.tsx

```diff
@@ -406,4 +516,55 @@ export class TabNav {
       tabTitles[this.selectedTabId].focus();
     });
   }
+
+  private getOverflowIcons(): VNode | VNode[] {
+    const { messages } = this;
+    const tabNavWidth = this.el.offsetWidth;
+
+    const tabTitles = Array.from(this.el.querySelectorAll("calcite-tab-title"));
+
+    const firstTitle = tabTitles[0].getBoundingClientRect();
+    const lastTitle = tabTitles[tabTitles.length - 1].getBoundingClientRect();
+
+    const isOverflowingRight = lastTitle.right > tabNavWidth;
+    const isOverflowingLeft = firstTitle.left < 0;
+
+    const getActionChevronDirection = (overflowDirection: string): VNode => {
```

Comment:
nitpick: should `overflowDirection` be typed to `"left" | "right"`? Maybe it should be `start` and `end`?